### DISCUSSION TITLE
[Test] A new type of objectives: Personal Objectives

### DISCUSSION THREAD

**Discussion by lucca65** [cambiatus/frontend#798] [Proposals]
## Context 

Today, one of our main innovations is in the way we generate new tokens. Instead of an arbitrary "mint", we rely on "objectives", a simplified version of Value Based Decision Making. The objective is for the entire community and it can be done by any member.   During our CMBGO and CMBX co-design sessions something came to me: basically all actions on those communities aim to help our members grow and learn.   The problem is that we have no ways of knowing exactly how well are our members learning. We can measure their CMBGO, but that alone won't give us specifics on what exactly already did and need to work on. This is why I think we can create a variation of our community objectives that I call Cambiatus Journey.     

## Ideas / Usage 

Every objective is for all members, but they completion is individualI may have completed 2/3 actions on an objective and another member only 1/3. This way we can track and show what is that our users can do still.
Actions would still be the way we generate coins but now we would have member tracking on what they done.
After I've completed an objective I know hold a certain knowledge and thus I'm entitled to a role or any other way we can show what the user know.

**Some examples**: Financial Resilient, Decision Maker, Cambiatus Artisan, Platform Helper, Co-design contributor, etc

This new type of objective will not replace the old one, it's an additional way. Several people working on their own growth now can be measured as community growth. Collective building but in a personal level. Our current structure is good for collective action, but not so for personal action  

For example:   
- **Objective**: Members understand the basis of Value Based Decision Making 
- **Goal**: 50 CMBGO or 5 actions 
- **Reward**: Role Financial Resilient   

With that set we can begin to think on the implementation.    

## Resources 

We simply can define those for now, adding it as a part of our co-design with communities. To start we can simply have this a way to learn how this can be used and if it really helps members to understand their collective intelligence and participation   

## Objective

Keep improving Decision Making and personal growth in a collective manner

**Comment by henriquecbuss** [comment]
Will these actions also be voted on by validators? If not, will there be a way to know if the user really did the action?

**Comment by henriquecbuss** [comment]
Will these actions have token rewards as well?
